src/pages/Filaments.tsx

```diff
@@ -5,8 +5,12 @@ import { Link } from "react-router-dom";
 import Breadcrumb from "../components/Breadcrumb";
 import EmptyResult from "../components/EmptyResult";
 import Section from "../components/Section";
+import { useState } from "react";
+import ConfirmationDialog from "../components/ConfirmationDialog";
 
 const Filaments = () => {
+  const [isDialogOpen, setIsDialogOpen] = useState(false);
+  const [deletingFilament, setDeletingFilament] = useState<any>();
```

Comment:
Use the correct type here:

```suggestion
  const [deletingFilament, setDeletingFilament] = useState<Filament>();
```

Reply:
Done 🚀